Plugins/Gource/GourcePlugin.cs

```diff
@@ -249,9 +247,9 @@ private static string SearchForGourceUrl()
                     return "https://github.com" + match.Groups[1].Value;
                 }
 
-                response = webClient.DownloadString(@"https://github.com/acaudwell/Gource/releases/tag/gource-0.42");
+                response = webClient.DownloadString(@"https://github.com/acaudwell/Gource/releases/tag/gource-0.47");
 
-                regEx = new Regex(@"(?:<a .*href="")(.*gource-.{3,15}win32\.zip)""");
+                regEx = new Regex(@"(?:<a .*href="")(.*gource-.{3,15}win64\.zip)""");
```

Comment:
we are still supporting x86 platform, so this may cause problems.

Reply:
This is just the default download, you can always download yourself too.
This change is not important, I rather keep the old code and drop this than try to make a full supporting change.